src/Administration/Resources/app/administration/src/core/factory/api-context.factory.js

```diff
@@ -75,5 +75,5 @@ function getAssetsPath(installationPath, isDevMode) {
         return '/bundles/';
     }
 
-    return `${installationPath}/bundles/`;
+    return `${installationPath}/bundles/administration/`;
```

Comment:
Thinking about it again, I guess this part of the path should rather be moved to the `assetFilter` calls instead to not mess with extensions. Paths would look like the following in practice.

`/bundles/swagmigrationassistant/administration/static/` or `/bundles/administration/administration/static/`

@jleifeld Can you maybe shed some light on why the path's changed (presumably) with the Vite migration? Wasn't involved, so I'm lacking the context here.